client/src/assets/images/icons/index.js

```diff
@@ -1,425 +1,164 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 
-export const CohortIcon = () => (
+const Icon = ({ children, ...props }) => (
```

Comment:
No need for children extraction, if you spread props over the component it will render children as well.